src/initializers/bull/index.ts

```diff
@@ -0,0 +1,51 @@
+import { list, decode } from './bull-helper';
+import { start } from './bull';
+import { getStats, reportStats } from './bull-metrics';
+import { parseURL } from 'ioredis/built/utils';
+import defaults = require('lodash/defaults');
+import { getLogger } from '../log4js';
+import * as cron from 'node-cron';
```

Comment:
Shall we make this dependency also optional? @nikostoulas

Reply:
This and bull could be made optional. This seems light with not many dependencies. So it might not worth it. Bull on the other hand might worth it. I don't have any strong opinion. 
@MariaEmma note that if they are not optional their vulnerabilities will affect all our projects.

Reply:
You are right, we should make it optional, I will do the changes.